cache/shared/helpers.go

```diff
@@ -62,14 +62,45 @@ func PushAllDevices(c Cache) {
 
 }
 
+func updateHeartbeat(v events.Event, c Cache) {
+	split := strings.Split(v.GeneratingSystem, "-")
+	if len(split) < 3 {
+		if strings.Contains(strings.ToUpper(v.GeneratingSystem), "AWS") {
```

Comment:
since the only difference with this if block is the log statement, let's just get rid of the AWS check